pkg/ccl/backupccl/restore_planning.go

```diff
@@ -2079,12 +2079,6 @@ func doRestorePlan(
 		fromDescription = from
 	}
 
-	if restoreStmt.Options.ExperimentalOnline {
-		if err := checkRewritesAreNoops(descriptorRewrites); err != nil {
-			return err
-		}
-	}
```

Comment:
🎉 I think we can probably remove the `checkRewritesAreNoops` function for now as well?